packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/packages.config

```diff
@@ -1,5 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Microsoft.Net.UWPCoreRuntimeSdk" version="2.2.10" targetFramework="native" />
   <package id="Microsoft.UI.Xaml" version="2.3.191129002" targetFramework="native" />
   <package id="Microsoft.Windows.CppWinRT" version="2.0.200316.3" targetFramework="native" />
-</packages>
+  <package id="runtime.win10-arm.Microsoft.Net.UWPCoreRuntimeSdk" version="2.2.10" targetFramework="native" />
```

Comment:
>package [](start = 3, length = 7)

It seems to be the missing part in my PR: [https://github.com/microsoft/react-native-windows/pull/4982](https://github.com/microsoft/react-native-windows/pull/4982).

Reply:
@vmoroz  that's because you haven't merged that change yet ;-)